charts/comet-site-v1/values.yaml

```diff
@@ -118,10 +118,9 @@ ingress:
   annotations:
     cert-manager.io/cluster-issuer: "letsencrypt"
 
-pvc:
-  create: false
-  annotations: {}
-  additionalPodLabels: {}
+domains: [] # one of these 3 has to be set for ingress to work
+domain: "" # one of these 3 has to be set for ingress to work
```

Comment:
Why the difference between domain and domains? Can't we harmonise that?

Reply:
Domain is for legacy reasons. I added a comment to explain it